config/locales/en.yml

```
@@ -37,8 +37,13 @@ en:
   attributes:
     search_stat:
       ad_count: Ad count
+      ad_count_top: Ad count top
       keyword: Keyword
       link_count: Link count
+      non_ad_count: Non ad count
       raw_response: Raw response
       total_result_count: Total result count
       user_id: User
+  buttons:
+    search_stat_details: Show Details
+    raw_response: View Raw Response
```

Alphabetical order ✨ 
```suggestion
    raw_response: View Raw Response
    search_stat_details: Show Details
```

---

Fixed in 7ea292090c3620e45e3159d7233806e5f545ec12